docs/technical-documentation/migrate-csv.md

```diff
@@ -90,6 +90,7 @@ From your `islandora-playbook` directory, issue the following commands to enable
 - `cd /var/www/html/drupal/web/modules/contrib` to get to your modules directory.
 - `git clone https://github.com/Islandora/migrate_islandora_csv` to clone down the repository from GitHub.
 - `drush en -y migrate_islandora_csv` to enable the module, installing the migrations as configuration.
+- `drush fim migrate_islandora_csv` to import the feature.
```

Comment:
Doing a feature import right after installation won't do anything (as the installation already added the current state of the configs).

Instead, we use the `drush fim` command to push config _changes_  into Drupal.

Reply:
Perhaps we need a new heading of "Making Changes to Migrations" where we note both the Drush features:import command (as we demonstrate here) and using the Configuration Synchronization UI for single item updates.

Reply:
I don't think I understand that part well enough to write it, but I'm happy to make the changes if some one can give me the words 😄

Reply:
From the list, this didn't work anyway. https://groups.google.com/g/islandora/c/AQaYVj46Q00/m/8RW7gSfuBQAJ